pulp_2to3_migration/tests/functional/plans/base.py

```diff
@@ -0,0 +1,3 @@
+from collections import namedtuple
+
+TestSetup = namedtuple('TestSetup', ['plan', 'outcome'])
```

Comment:
I'm not sure I understand the added value of this extra layer of indirection.  Why not just feed these arguments directly into ```_load_and_run()```?

Personally I would probably not use ```mp_validation.py``` either, and just leave them defined in ```test_file_migration.py```

Reply:
The attempt was to make it more readable and define plan and expected result in one place, without coming back and forth looking at the plan and figuring out what the result should be for each test.

It might not be the best way. I can put everything into one file all together. However I'd like to reuse some plans, so I thought a place for constants would be good. Throwing all plans into one file looks too much to me, so I transitioned to grouping plans in some way, in some logical(?) way. Currently the logic is "for MP validation tests, put all plans in a dedicated file, for the other set of tests, put the plans in the other file".

Is putting plans in the same file as tests and having the expected results in each test a preferable option? I can do that, no problem.
I'm also open for any third option to approach that.

Reply:
I'm not strongly against it.  If you plan to use these constants in multiple files then it could make sense -- it just looked like these constants were only meant for the tests in this one file.

Reply:
If the tests are being reused (the named tuple things), in what way would they be tested differently each time?

Reply:
tuples are unique, plans are reused.